### ISSUE TITLE
Assemble list of Pros & Cons of Tribeca CMS

### ISSUE THREAD

**Issue by kylejbertelsen** [akcdotorg/akc.org#142]

**Comment by stevepessah** [comment]
@kylejbertelsen @spaulenoff do you want me to kick this project off? Did they deliver a feature set for their platform?

**Comment by kylejbertelsen** [comment]
@stevepessah: they promised that they'd deliver a demo, but we haven't seen that yet. I'll withhold my judgment on this until I've had a chance to dig around in it.

**Comment by kylejbertelsen** [comment]
Possible concerns, which might be able to be cleared-up by them answering a few questions:
- Workflow -- they spoke of a delay of a day or more in getting our code from our development environments into the development servers, staging servers, etc. If that is the case, I think it's a dealbreaker, since we need to be able to move relatively fast
- Black-boxing -- If, as it sounded from our one meeting with them, they do not share with us the code behind their custom modules, that handicaps us in terms of what we can accomplish. Perhaps an NDA would be enough to open up their black boxes?
- Lock-in -- If we should go with them, even in the short-term, are we then locked-in with them? If there's much of their Drupal work that is considered proprietary, we wouldn't have the freedom to move past them in the future. One of the benefits of going to an open-source CMS like Drupal is the flexibility we'd have to control our own destiny. Would working with Tribeca handcuff that flexibility? As an agency feeding at the open-source table, it's a little troublesome that they'd have black boxes at all: part and parcel of being a part of such a community is that people who eat at the trough also give back to the community.
- What do these proprietary customizations do, anyway? It's entirely possible that what they're offering is no more than preconfigurations and assembly of freely-available modules. I want to see a description of what they are, and what they do.
- Control -- Would we have unfettered access to the administration of the CMS? I can easily see how we'd want to have the ability to install and configure our own choices of (or creation of) Drupal modules. Is that allowed in their setup?
- Hosting -- As I understand it, Tribeca is also a webhost. Should we find a better webhost, would we have the ability to move to that, or are we locked-in as both a platform AND a webhost?
